_assets/gulp_config/paths.js

```diff
@@ -68,4 +68,22 @@ paths.jekyllImageFilesGlob = paths.jekyllImageFiles + paths.imagePattern;
 // Site files globs
 paths.siteHtmlFilesGlob = paths.siteDir + paths.htmlPattern;
 
+// HTML pages to run through the accessibility test.
+paths.htmlTestFiles = [
+  '_site/index.html',
+  '_site/2017/**/*.html',
+  '_site/blog/index.html',
+  '_site/blog/tag/drupal/index.html',
+  '_site/company/index.html',
+  '_site/company/anne-tomasevich/index.html',
+  '_site/company/mission-and-values/index.html',
+  '_site/contact/index.html',
+  '_site/results/index.html',
+  '_site/results/case-studies/index.html',
+  '_site/results/case-studies/duke-today/index.html',
+  '_site/results/open-source/index.html',
+  '_site/services/index.html',
+  '_site/services/development/index.html'
+];
+
```

Comment:
At first I was testing everything via wildcards but the test was taking 7-15 minutes depending on what I included. I don't think we need to test every tag page or team member page or old blog post. I thought carefully about which files needed to be tested and I think this list makes sense - it captures landing pages like home or company, one team member page, one case study page, etc., and recent blog posts.

We need to test files repeatedly if we update features or styles. But it doesn't make sense to test everything when, say, someone adds a new blog post - only that post needs to be tested. I guess I need to rethink the strategy.